pkg/vsphere/toolbox/backdoor.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
This file does compile and run on Darwin as well, tho if you run cmd/toolbox/main.go it'll bail with an error "not in a virtual world" - but in theory that wouldn't happen if run within a VM.  Likewise for Windows.